base/stat.jl

```diff
@@ -460,6 +460,12 @@ islink(path...) = islink(lstat(path...))
 
 # samefile can be used for files and directories: #11145#issuecomment-99511194
 samefile(a::StatStruct, b::StatStruct) = a.device==b.device && a.inode==b.inode
+
+"""
+    samefile(path_a::AbstractString, path_b::AbstractString)
+
+Check if the paths `path_a` and `path_b` refer to the same file.
```

Comment:
What about "...the same _existing_ file _or directory_"?

Reply:
If you can make this as a suggestion, it will be easy to merge.

Reply:
```suggestion
Check if the paths `path_a` and `path_b` refer to the same existing file or directory.
```